docs/DeveloperGuide.md

```diff
@@ -695,22 +726,103 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
+--------------------------------------------------------------------------------------------------------------------
+
+### Adding an applicant
+1. Adding an applicant to TrackAScholar while all applicants are displayed in GUI.
+   1. Prerequisites: List all applicants using the `list` command. Multiple applicants in the list. The list contains 
+      an applicant named `Alex Yeoh` but does not contain an applicant named `Benjamin low`.
+   
+   1. Test case: 
+      ```
+      add n/Benjamin low 
+      p/98765431 
+      e/benLow@gmail.com 
+      s/NUS Global Merit Scholarship 
+      as/pending 
+      m/Medicine 
+      m/Mathematics
+      ```
+      Expected: An applicant named `Benjamin Low` with the above characteristics
```

Comment:
Capitalization inconsistent for `Benjamin Low` and `Benjamin low` in the previous few lines.